docs/developer/release-procedures.md

```diff
@@ -1,6 +1,35 @@
 # Release procedures
 
-___It will be moved soon, for now you can [visit Wiki](https://github.com/kubernetes/dashboard/wiki/Release-procedures).___
+## Official releases
```

Comment:
There is a step (or two) missing that involves building and pushing the container image to the official repo(s) /cc @maciaszczykm @floreks

Reply:
I could not find the procedures to build and push container image for official release. Could you tell me the procedures or follow up this PR?

Reply:
Right now there are no manual steps required to push the images. Everything is done by the Travis after successful TAG build.